recipe/meta.yaml

```diff
@@ -15,6 +15,7 @@ build:
 requirements:
   build:
     - python
+    - setuptools
```

Comment:
This may not be needed. Instead, we use `pip` command to install things, as recommended by the conda-forge team: https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip.

Reply:
I can't write comments to the specific lines above. Here are a couple of suggestions:
- parametrize the name of the package
- remove the `fn: ...` line in the `source` section
- add the corresponding `sha256` value for the archive
- include the LICENSE file and add a corresponding field in the `about` section

See https://github.com/nsls-ii-forge/nslsii-feedstock/blob/master/recipe/meta.yaml as an example of a `meta.yaml` file.

Reply:
One more thing: they now recommend to name the `build` section as `host`. Please do this renaming as well.

Reply:
> This may not be needed. Instead, we use `pip` command to install things, as recommended by the conda-forge team: https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip.

Before, without setuptools, it failed.
https://dev.azure.com/nsls2forge/nsls2forge/_build/results?buildId=2412&view=logs&j=1bf226d3-0e2f-52d8-fa93-7d9e633347b3&t=ec8d466d-e3b4-5115-4f06-222398f91e6c&l=423

Reply:
There should be a `pip` host dependency, with the updated build command:
```yaml
  script: "{{ PYTHON }} -m pip install . --no-deps -v"
```